### ISSUE TITLE
QA: Proposal state identifiers not matching figma

### ISSUE THREAD

**Issue by 0xTranqui** [public-assembly/flexible#114]
tldr all of the proposal state identifiers on the card are solid black backgrounds atm, but at least "pending" is supposed to have a transparent background. makes it confusing as all of the states look the same from a cursory glance. see screenshots from Flexible + figma for comparison:

flexible:
<img width="1341" alt="Screen Shot 2023-04-04 at 8 26 38 AM" src="https://user-images.githubusercontent.com/93691906/229791038-8340536c-ded5-4c1c-bc79-912532c74a67.png">

figma:
<img width="955" alt="Screen Shot 2023-04-04 at 8 27 34 AM" src="https://user-images.githubusercontent.com/93691906/229791254-d1625fa6-13c5-454f-ab03-65c4b7b6ddf1.png">
